src/graph/graph.h

```diff
@@ -329,6 +339,9 @@ GrB_Matrix Graph_GetTransposedRelationMatrix(
 // internal matrices, caller mustn't modify it in any way.
 GrB_Matrix Graph_GetZeroMatrix(const Graph *g);
 
+// Returns wether the matrix contains multi edge.
+bool Graph_RelationshipContainsMultiEdge(const Graph *g, int r);
+
```

Comment:
Move to line 324 right below `Graph_GetRelationMatrix`
Do the same in the `graph.c` file